client/my-sites/plugins/plugin-install-button/index.jsx

```diff
@@ -195,6 +199,38 @@ export class PluginInstallButton extends Component {
 		);
 	}
 
+	renderIncompatiblePluginNotice() {
+		const { translate, isEmbed } = this.props;
+		return (
+			<div className={ classNames( { 'plugin-install-button__install': true, embed: isEmbed } ) }>
+				<span
+					onClick={ this.togglePopover }
+					ref="disabledInfoLabel"
+					className="plugin-install-button__warning"
+				>
+					{ translate( 'Incompatible Plugin' ) }
+				</span>
+				<InfoPopover
+					position="bottom left"
+					popoverName={ 'Plugin Action Disabled Install' }
+					gaEventCategory="Plugins"
+					ref="infoPopover"
+					ignoreContext={ this.refs && this.refs.disabledInfoLabel }
+				>
+					<div>
+						<p>{ translate( 'This plugin is not supported on WordPress.com.' ) }</p>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Incompatible plugin: This plugin is not supported on WordPress.com.' )` *ES Score: 6*